app/Console/Commands/BeginQueueAllModelsCommand.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Console\Commands\Humidity\HumidityBeginQueueCommand;
+use App\Console\Commands\Temperature\TemperatureBeginQueueCommand;
+use Illuminate\Console\Command;
+
+
+class BeginQueueAllModelsCommand extends Command
+{
+    protected $signature = 'models:beginQue';
```

Comment:
```suggestion
    protected $signature = 'models:beginQueue';
```
this feels like you want to "beginWhat" because Que is italian/spanish for what. the word you're looking for is queue :)